lib/fastlane/plugin/firebase_test_lab/options.rb

```diff
@@ -75,7 +75,13 @@ def self.available_options
                                        verify_block: proc do |value|
                                          v = File.expand_path(value.to_s)
                                          UI.user_error!("Key file not found at path '#{v}'") unless File.exist?(v)
-                                       end)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :ios_xc_test_args,
```

Comment:
Creating a single param for the xcode version seems enough? We don't want to expose the structure of the request payload if not necessary.

Reply:
Theres other params that can be added in xc_test_args, seems like a waste of time having individual params for each one.